src/foam/nanos/crunch/MinMaxCapabilityData.js

```diff
@@ -31,21 +31,9 @@ foam.CLASS({
       name: 'selectedData',
       class: 'StringArray',
       javaFactory: 'return new String[0];',
-      adapt: function(_, n){
-        if ( foam.Array.isInstance(n) ) return n;
-        if ( foam.core.String.isInstance(n) ){
-          return [
-            n
-          ];
-        }
-
-        if ( foam.Function.isInstance(n.toString) ) {
-          return [
-            n.toString()
-          ];       
-        }
-
-        return n;
+      adaptArrayElement: function(o, prop) {
```

Comment:
So the adaptArrayElement defined in types always returns a string?

Reply:
yes the element of StringArray should always be string.